admin/src/components/Avatar/index.tsx

```diff
@@ -12,16 +12,23 @@ import PropTypes from "prop-types";
 import { isObject } from "lodash";
 import { Avatar, Initials } from "@strapi/design-system/Avatar";
 import { renderInitials } from "../../utils";
+import { IntlContext } from "react-intl";
+import AdminAvatar from "../AdminAvatar";
 
-const UserAvatar = ({ avatar, name }) => {
+const UserAvatar = ({ avatar, name, isAdminComment }) => {
```

Comment:
You have removed `isAdminComment` from the PropTypes but did not introduce the types for the props. That was not my intention. Could we remove all propTypes and introduce types like: 
```suggestion
interface IProps {
    avatar: string | ToBeFixed;
    name: string;
    isAdminComment?: boolean;
};

const UserAvatar: React.FC<IProps> = ({ avatar, name, isAdminComment = false }) => {
```
Let me know what you think. 😁